"experience_privateserverlinks" translated (pt_br)

S1lB opened the pull request:
This Pull Request translates the new content into Portuguese

S1lB committed:
"experience_privateserverlinks" translated (pt_br)

CarBluGStudio commented on locales/pt_br/messages.jsonc:
```diff
@@ -276,185 +276,185 @@
     },
     "experience_privateserverlinks_importprivateserversmodal_importinfo": {
-        "message": ""
+        "message": "Você gostaria de importar os seguintes {total} servidores privados:"
```
Update according to Jullian's fix on singular and plural

CarBluGStudio commented:
Jullian just fixed the line 339, I think it's good to fix it here too

It would look something like this
 "Você gostaria de importar o(s) seguinte(s) {total, plural, =1 {servidor privado} other {# servidores privados}}:"

S1lB committed:
Fix

sorry for the mess lol

S1lB commented:
> Jullian just fixed the line 339, I think it's good to fix it here too
> 
> It would look something like this
>  "Você gostaria de importar o(s) seguinte(s) {total, plural, =1 {servidor privado} other {# servidores privados}}:"

I just fixed it, thanks for letting me know :)

S1lB committed:
Merge branch 'RoSeal-Extension:main' into main

S1lB committed:
New commit translated

I didn't translate the title and description because CarBluGStudio already did.
